### ISSUE TITLE
flux jobspec command

### ISSUE THREAD

**Issue by garlick** [flux-framework/flux-core#1929]
@grondo and I were just discussing: it would be nice to have an installed "plumbing" command to emit jobspec, like @SteVwonder 's `emit-jobspec.py` test script.  Maybe it could emulate familiar environments, like oar and slurm?  E.g.
```
$ flux jobspec srun -N2 -n2 | flux job submit
```
or
```
$ flux jobspec oarsub -l /nodes=12/cpu=2 | flux job submit
```
(but just start out as a "promotion" of the above script, with the subcommand structure and `srun` subcommand added)?

Edit: dropped the start time reservation from oarsub example!  Keep it simple.

**Comment by SteVwonder** [comment]
Sounds like a reasonable next-step.  I was looking for something to do on the flight home.  If no one else is already working on this, I volunteer to take it on.

**Comment by garlick** [comment]
Closed by #1964
